wof/core.py

```diff
@@ -49,7 +49,8 @@
 except ImportError:
     version = 'dev'
 
-curr_folder = os.path.abspath(os.path.dirname(__file__))
+_root_dir = os.path.abspath(os.path.dirname(__file__))
+_TEMPLATES = os.path.join(_root_dir, 'app', 'templates')
```

Comment:
They exist in both locations :unamused: 

```
wof/apps/templates:
total 156K
-rw-r--r-- 1 filipe users 2.0K Jan 18 07:29 index_1_0.html
-rw-r--r-- 1 filipe users 2.5K Jan 18 07:29 index_1_1.html
-rw-r--r-- 1 filipe users 1.3K Jan 18 07:29 index_2.html
-rw-r--r-- 1 filipe users  948 Jan 18 07:29 index.html
-rw-r--r-- 1 filipe users  14K Jan 18 07:29 wml2_values_template.xml
-rw-r--r-- 1 filipe users  73K Jan 18 07:29 wsdl_1_1_template.wsdl
-rw-r--r-- 1 filipe users  48K Jan 18 07:29 wsdl_temp.wsdl

wof/flask/templates:
total 156K
-rw-r--r-- 1 filipe users 2.0K Jan 18 07:29 index_1_0.html
-rw-r--r-- 1 filipe users 2.5K Jan 18 07:29 index_1_1.html
-rw-r--r-- 1 filipe users 1.5K Jan 18 07:29 index_2.html
-rw-r--r-- 1 filipe users 2.6K Jan 18 07:29 index.html
-rw-r--r-- 1 filipe users  13K Jan 18 07:29 wml2_values_template.xml
-rw-r--r-- 1 filipe users  73K Jan 18 07:29 wsdl_1_1_template.wsdl
-rw-r--r-- 1 filipe users  48K Jan 18 07:29 wsdl_temp.wsdl
```

But the I used the same location (`../../../wof/apps/templates`) that we had in the original code.